doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -563,6 +563,20 @@ for it.
 - `&x` as a Coq constr expression expands to
   `ltac2:(Control.refine (fun () => hyp @x))`.
 
+In the special case where Ltac2 antiquotations appear inside a Coq term
+notation, the notation variables are actually systematically bound in the body
```

Comment:
maybe remove "actually systematically" - doesn't add anything to the sentence, does it?